stdlib/public/core/StringObject.swift

```diff
@@ -436,6 +436,10 @@ extension _StringObject.Nibbles {
   internal static func largeCocoa(providesFastUTF8: Bool) -> UInt64 {
     return providesFastUTF8 ? 0x4000_0000_0000_0000 : 0x5000_0000_0000_0000
   }
+  
+  internal static func largeFastImmortalCocoa() -> UInt64 {
+    0xC000_0000_0000_0000
```

Comment:
As far as I can tell 1100 was an unused bit pattern and should have the desired meaning of "immortal, bridged"